projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -89,6 +90,47 @@ import { DataTableGhostLoaderComponent } from './ghost-loader/ghost-loader.compo
             >
             </datatable-summary-row>
           }
+          <ng-template
+            ngx-datatable-body-row
+            #bodyRow
+            let-groupRow="row"
+            let-rowIndex="index"
+            let-groupValue="groupValue"
+            let-disabled="disabled"
+            let-group="group"
```

Comment:
```suggestion
            let-row="row"
            let-rowIndex="index"
            let-groupValue="groupValue"
            let-disabled="disabled"
```

group seems to be unused, just calling row `row` is more intuitive